sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -957,8 +1022,8 @@
                                     "computeruntime": {
                                         "SparkRuntimeVersion": "[parameters('spark_runtime_version')]"
                                     },
-                                    "offlinestoreconnectionname": "[variables('offlineStoreConnectionName')]",
-                                    "onlinestoreconnectionname": "[if(not(empty(parameters('online_store_target'))), variables('onlineStoreConnectionName'), '')]"
+                                    "offlinestoreconnectionname": "[parameters('offline_store_connection_name')]",
```

Comment:
follow the comment above, will the connection name literally be "empty" in any case?

Reply:
it's not possible, in this case, parameters('offline_store_connection_name') can only be "" and it will be none in workspace document which means there is no workspace connection created.